data/api/client-server/relations.yaml

```
@@ -294,6 +297,27 @@ components:
       example: org.example.my_relation
       schema:
         type: string
+    recurse:
+      in: query
+      name: recurse
+      x-addedInMatrixVersion: "1.10"
+      required: false
+      description: |-
+        Whether to additionally include events which only relate indirectly to the
+        given event, ie. events related to the root events via one or more direct relationships.
+
+        If set to `false`, only events which have direct a relation with the given
+        event will be included.
+
+        If set to `true`, all events which relate to the given event, or relate to
+        events that relate to the given event, will be included.
+
+        It is recommended that at least 3 levels of relationships are traversed.
```

By whom? also, avoid passive voice.

```suggestion
        It is recommended that homeservers traverse at least 3 levels of relationships.
```